source/appController/AMDSClientAppController.h

```diff
@@ -88,7 +99,12 @@ public slots:
 	void openNetworkSession();
 
 	/// slot to handle socket error signal from the server
-	void onAMDSServerError(AMDSServer* server, int errorCode, const QString &socketKey, const QString &errorMessage);
+	void onAMDSServerError(const QString &serverIdentifier, int errorCode, const QString &socketKey, const QString &errorMessage);
+
+	/// slot to handle the signal to start a new scan
+	void onScanStarted();
+	/// slot to handle the signal to finish a scan
+	void onScanStopped();
 
```

Comment:
Hey @XuetaoLiu-CLS and @dretrex 
Where's the implementation for these functions? I can't seem to find it in the pull request.